### PR TITLE
Update 2020_11_14-pkt-community.md

### PR DESCRIPTION
**Pull request by josh-berger**
Here is the final milestone of the PKT Community Project.

### CODE REVIEW

**Review by cjdelisle — APPROVED** [review]
The high level project description is:

> This project is for communication, documentation and community building for the PKT community. The point of this project is to help people understand why and how they can participate in making the PKT global vision a reality. The key points of the project are:
Awareness: There is an alternative vision to the world of ISP monopolies
Importance: Why a person should care about free access to information
Documentation: How to navigate the ecosystem and technologies
Community: Maintaining a safe and positive environment for project participants

The no-pumping statement of the project proposal is as follows:

> It is inherent to any communication that people may take it as a sign to trade the coin, and we acknowledge that a higher trading price of PKT is beneficial for many aspects of the project. However, we believe that bringing in speculators is harmful to the project for multiple reasons including dilution of the project vision as well as unreasonable price volatility. Therefore our communications will be focused first and foremost on the vision of a better internet and how we can work together to achieve that.

--

From the simple perspective of "boxes checked", it is clear to me that the project owner has checked all of the boxes and then some - it is notable that [www-pkt-cash](https://github.com/pkt-cash/www-pkt-cash/commits/master/) has been continuously updated by coconut3-dev, who I know to be one of their colleagues.

I think the best way to evaluate the content of the submission is through the lens of the phrase "our communications will be focused first and foremost on the vision of a better internet and how we can work together to achieve that."

From that perspective, all five of the videos make logical sense. For my part, I feel there has been a positive evolution over the past years in the clarity and exactness of how the concepts are explained.

The animated video focuses on the idea of making money by sharing your internet with neighbors who are willing to pay for it and does a decent job of explaining oversubscription in layman's terms. I think the biggest challenge with this video is the fact that some ISPs forbid sharing of wifi and that fact is not discussed - but if there was a 90 second video teasing the idea that you can, for example, rent your home on AirBNB, I would not classify it as a severe deficiency if it mentioned that certain localities or home owners associations may forbid that. For what it's worth, I'm glad that the creators didn't opt for some ritualistic CYA statement along the lines of "conditions apply, check local regulations" because these type of statements don't bring any real value to the viewer. My opinion is that a better video could be made, but this video is not deficient.

The articles are all taken from the minutes of a community town-hall meeting where various community members discussed their initiatives and how they might collaborate. This activity of minuting these meetings and presenting the result is obviously valuable from a Documentation and a Community perspective, and I don't think there's really much to be said about them.

The technical articles are perhaps even more valuable than the articles because they provide a detailed synthesis of activities in all of the projects and initiatives which have taken place in the PKT ecosystem.

It's noteworthy that these articles are just a selection of the ones on https://crypto.pkt.cash/ which has been maintained throughout the past 3 years.

The graphics are described as "Graphics and animations for helping explain the inner workings of the technological components as well as explaining the ways in which a person can become a community member." I don't think they really lived up to this objective, particularly regarding the "explain the inner workings", but I'm ready to accept that effort was re-assigned from this task to others - e.g. WWW_UPDATE and TECH_ARTICLE which clearly went beyond what was specified in the project. I don't think there is a realistic basis for claiming that the project was not delivered.

WWW_DOCS, WWW_I18N, and WWW_UPDATE are obviously completed. Again of note, there are 6 translations while the project specified only 3.

All told, I think it is obvious that there was more effort done on this project than was necessary and there's really no reasonable way to argue that the project owner was "lazy" or somehow didn't pursue the project in good faith. Over the years, the NS has become significantly more sensitive to "communication-centric" projects, and I think the project owner has been quite receptive to NS feedback and has adapted accordingly. There may be arguments made about the quality or thoughtfulness of some of the work, but since it is already established that the project owner was not trying to do the minimum, there is no reasonable basis for rejecting a project milestone based on the claim that it "could have been better", so I think the only logical conclusion is that this milestone should be accepted and the project terminated successfully.

**Review by backupbrain — APPROVED** [review]
Verified. Everything looks good.
